docs/docs/intro.md

```diff
@@ -23,16 +23,16 @@ ZMK is currently missing some features found in other popular firmware. This tab
 | [Keyboard Codes](codes/#keyboard)                                                                                         | ✅  |    ✅     | ✅  |
 | [Media](codes/#media-controls) & [Consumer](codes/#consumer-controls) Codes                                               | ✅  |    ✅     | ✅  |
 | [Encoders](features/encoders)[^1]                                                                                         | ✅  |           | ✅  |
-| [OLED Display Support](features/displays)[^2]                                                                             | 🚧  |    🚧     | ✅  |
+| [Display Support](features/displays)[^2]                                                                                  | 🚧  |    🚧     | ✅  |
 | [RGB Underglow](features/underglow)                                                                                       | ✅  |    ✅     | ✅  |
-| One Shot Keys                                                                                                             | 🚧  |    ✅     | ✅  |
-| Combo Keys                                                                                                                | 🚧  |           | ✅  |
+| One Shot Keys                                                                                                             | ✅  |    ✅     | ✅  |
+| [Combo Keys](https://github.com/zmkfirmware/zmk/pull/504)                                                                 | 🚧  |           | ✅  |
 | Macros                                                                                                                    | 🚧  |    ✅     | ✅  |
 | Mouse Keys                                                                                                                |     |    ✅     | ✅  |
```

Comment:
I think @innovaker  may be working on this? So may be in dev now.

Reply:
It's a good idea, but I don't think we're looking to officially adopt a position on steno yet.  Perhaps if the steno devs/users band together as we've previously suggested to them (it needs someone new to lead that charge).